README.md

```diff
@@ -44,10 +44,10 @@ npm install @open-rpc/docs-react @open-rpc/meta-schema --save
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Documentation from "@open-rpc/docs-react";
-import { OpenRPC } from '@open-rpc/meta-schema';
+import { OpenRpcDocument } from '@open-rpc/meta-schema';
```

Comment:
Capitalize RPC?

Reply:
take it up with `@open-rpc/meta-schema`

Reply:
Ah, nm, generated code.

Reply:
lol quick res, didn't even load ya before my follow up ;)
yeap.